eclair-node/src/main/resources/application.conf

```diff
@@ -38,6 +38,10 @@ eclair {
   fee-base-msat = 546000
   fee-proportional-millionth = 10
 
+  // mamixmum local vs remote feerate mismatch; 1.0 means 100%
+  // actuel check is abs((local feerate - remote fee rate) / (local fee rate + remote fee rate)/2) > fee rate mismatch
```

Comment:
I think that comment should rather be in the scaladoc of function isFeeDiffTooHigh